test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js

```diff
@@ -11,13 +13,23 @@ const mockWithoutIdentify = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(identifyWithIdpsNoIdentify);
 
-fixture(`Identity With IDPs Form`);
+const mockOnlyOneIdp = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(identifyOnlyOneIdp)
+  .onRequestTo('http://localhost:3000/sso/idps/facebook-idp-id-123?stateToken=inRUXNhsc6Evt7GAb8DPAA')
+  .respond(async (req, res) => {
+    // Delay response for 2 seconds
+    await new Promise((r) => setTimeout(r, 2000));
```

Comment:
is this simulating the auto re-direct ?

Reply:
yeah but main goal is to slow down the redirect url so that can assert on SuccessRedirect view.